src/data/menu-guides.yaml

```
@@ -13,6 +13,9 @@
 - title: Send custom indicators
   link: /guides/slo/sending-custom-indicators/
   sub: true
+- title: Registering webhooks
```

Should this be:

```suggestion
- title: Registering Webhooks
```

As we capitalise it in the actual title of the post?

---

Yep